_posts/2017-06-07-announcing-fastruby-io-productized-rails-upgrades.md

```diff
@@ -0,0 +1,23 @@
+---
+layout: post
+title:  "Announcing FastRuby.io"
+date: 2017-06-07 15:06:00
+categories: ["rails", "rails upgrades"]
+author: "etagwerker"
+---
+
+Today we are happy to announce the launch of our first productized service: [fastruby.io](https://fastruby.io) - Speedy Ruby on Rails Upgrades by Ombu Labs.
+
+We are quite familiar with this sort of projects and decided to package it under its own domain. We have been doing [Ruby on Rails](http://rubyonrails.org) upgrades since 2009, for our [own products](https://www.ombulabs.com/#products) and [client projects](https://www.ombulabs.com/#clients), and we are looking forward to shipping more of them.
+
+<!--more-->
+
+If you have been using [Rails](http://rubyonrails.org) for a while, you know that big version jumps are hard. If you have a big Rails [monolith](https://martinfowler.com/bliki/MonolithFirst.html) with 100+ dependencies, you know that [upgrading Rails](https://fastruby.io) is not a trivial project.
+
+You know you need to keep your Rails app **up to date**, but it's usually *not* a priority. So you postpone upgrading until your version of Rails is **no longer supported**. Then you upgrade.
+
+More often than not, you start a [Rails upgrade](https://fastruby.io) branch, start bumping versions, find problems, troubleshoot them, and eventually get sidetracked by your business. Then you switch to another branch and the [Rails upgrade](https://fastruby.io) branch grows stale.
+
+Even the best teams have this problem! (e.g. [Github](http://shayfrendt.com/posts/upgrading-github-to-rails-3-with-zero-downtime/))
```

Comment:
I read an interview (https://blog.rubyroidlabs.com/2017/05/patterson) the other day and they're still stuck on Rails 3!

```
– Let’s talk about GitHub. Is it public, the version that Rails are running?
– Yes, it is public. Right now we are running Rails 3.2 in production, we are in the middle of upgrading to 4 and eventually getting on to edge. So I assume we will be up to 4.2 fairly soon within the next couple months or so. I think we have like 10 failing tests left. After so we are very close.
```